language/en/units.json

```diff
@@ -1331,9 +1331,9 @@
 			"corcomlvl10": "Specialized in frontline warfare",
 			"corcrash": "Amphibious Anti-air Bot",
 			"corcrus": "Cruiser",
-			"corcrw": "Flying Fortress from the past.",
+			"corcrw": "Old generation of Dragon Flying Fortress",
```

Comment:
???
Why is that pertinent at all?

Reply:
Fiiine i'll just copy description from normal Dragon